perfkitbenchmarker/flag_util.py

```diff
@@ -190,3 +191,149 @@ def DEFINE_integerlist(name, default, help, on_nonincreasing=None,
   serializer = IntegerListSerializer()
 
   flags.DEFINE(parser, name, default, help, flag_values, serializer, **args)
+
+
+SI_SUFFIXES = ['', 'K', 'M', 'G', 'T', 'P']
+
+
+@functools.total_ordering
+class ObjectSize(object):
+  """Holds the size of data, i.e. numbers of bytes.
+  """
+
+  def __init__(self, bytes=0):
+    self.bytes = bytes
+
+  def mult_for_base_and_exp(self, base, exp):
```

Comment:
Casing is odd. This is not a property. It looks like a private method.

The name is a bit confusing to me, but the docstring was enough for me to understand, so I'm not picky about it.